lib/chartlib.sh

```diff
@@ -378,8 +378,11 @@ chartlib::install_chart_with_all_configs() {
     local random_suffix
     random_suffix=$(tr -dc a-z0-9 < /dev/urandom | fold -w 16 | head -n 1)
 
+    local release_trimmed
+    release_trimmed=$(echo "$release"-"$random_suffix" | cut -c-29)
```

Comment:
Namespaces can be 63 characters long. Why the cut at 29?

Reply:
check the helm/charts#7320 issue happened

Reply:
```
$ helm ls
artifactory-ha-7219pws7otxtcr6g
$ kubectl describe -n artifactory-ha-dmn0miwywb4tcegj statefulset
...
  Warning  FailedCreate      3m (x25 over 8m)  statefulset-controller  create Pod artifactory-ha-dmn0miwywb4tcegj-artifactory-ha-member-0 in StatefulSet artifactory-ha-dmn0miwywb4tcegj-artifactory-ha-member failed error: Pod "artifactory-ha-dmn0miwywb4tcegj-artifactory-ha-member-0" is invalid: metadata.labels: Invalid value: "artifactory-ha-dmn0miwywb4tcegj-artifactory-ha-member-545fd89668": must be no more than 63 characters
...
```

Reply:
Wouldn't this be better if the `namespace` were a shorter name rather than things within that namespace? It would make identifying links easier.

Reply:
`namespace` is not the problem there, StatefulSet pod label `controller-revision-hash:` gets 65 characters long if there is no 29 characters limit put in for the release name.

Reply:
ok it is related to Kubernetes bug https://github.com/kubernetes/kubernetes/issues/64023